site/docs-md/apis/splash-screen/index.md

```diff
@@ -1,4 +1,4 @@
-<plugin-api index="true" name="splash-screen"></plugin-api>
+<plugin-platforms platforms="pwa,ios,android"></plugin-platforms>
```

Comment:
plugin-api is a different tag user for other thing, don't remove it
as there is no plugin-platforms tag, just add it, but without removing the other one.
Also, electron seems to be supported on electron too.